src/main.js

```diff
@@ -1 +1,5 @@
-// Write your code here
\ No newline at end of file
+// Write your code here
+console.log(2 + 2)
+console.log(3 * 3 * 3)
+console.log("Hello tornike")
+console.log("Hel" + "lo!")
```

Comment:
მხოლოდ ასოები უნდა გამოიყენო!
```suggestion
console.log("Hel" + "lo")
```